src/tests/subcommand_test.go

```diff
@@ -0,0 +1,77 @@
+package tests
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+// runAndAssertSubcommand executes a subcommand of the "semantifly" command with the provided arguments
+// and asserts that the output contains the specified assertion statement.
+//
+// Parameters:
+//   - subcommand (string): The subcommand to execute.
+//   - assertStatement (string): The expected substring to be present in the command output.
+//   - args ([]string): Additional arguments to pass to the subcommand.
+func runAndAssertSubcommand(subcommand string, assertStatement string, args []string) error {
```

Comment:
nit: assertStatement -> wantedStdoutSubstr

We are not actually directly asserting that arg, and "got"/"want" are golang naming conventions for unit test assertion args; this is a "want" variable

Reply:
Got it